### PR TITLE
[baseline] clean up and switch to keyword Supports

### PR DESCRIPTION
**Pull request by JackBoosY**
Clean up the baseline to prevent overdue additions from being missed.

### CODE REVIEW

File: ports/asyncplusplus/CONTROL [hunk 1]
```diff
@@ -1,3 +1,5 @@
 Source: asyncplusplus
-Version: 1.0-1
+Version: 1.0
+Port-Version: 2
 Description: Async++ is a lightweight concurrency framework for C++11
+Supports: !(arm&uwp)
```
Comment (JackBoosY) [inline ports/asyncplusplus/CONTROL:5]:
```suggestion
Supports: !uwp
```

File: ports/avro-c/CONTROL [hunk 2]
```diff
@@ -1,6 +1,7 @@
 Source: avro-c
-Version: 1.9.2-1
-Supports: !(uwp|osx)
+Version: 1.9.2
+Port-Version: 2
 Homepage: https://github.com/apache/avro
 Description: Apache Avro is a data serialization system
 Build-Depends: jansson, liblzma, zlib, snappy
+Supports: !(uwp|osx|(arm&uwp))
```
Comment (JackBoosY) [inline ports/avro-c/CONTROL:7]:
```suggestion
Supports: !(uwp|osx)
```

File: ports/benchmark/CONTROL [hunk 3]
```diff
@@ -1,5 +1,6 @@
 Source: benchmark
 Version: 1.5
+Port-Version: 1
 Homepage: https://github.com/google/benchmark
 Description: A library to support the benchmarking of functions, similar to unit-tests.
-Supports: !uwp
\ No newline at end of file
+Supports: !(arm&uwp)
```
Comment (JackBoosY) [inline ports/benchmark/CONTROL:6]:
```suggestion
Supports: !(arm|uwp)
```

File: ports/berkeleydb/CONTROL [hunk 4]
```diff
@@ -1,6 +1,6 @@
 Source: berkeleydb
 Version: 4.8.30
-Port-Version: 5
+Port-Version: 6
 Homepage: https://download.oracle.com/
 Description: BDB - A high-performance embedded database for key/value data.
-Supports: !(uwp|linux|osx)
+Supports: !(uwp|linux|osx|(arm&uwp))
```
Comment (JackBoosY) [inline ports/berkeleydb/CONTROL:6]:
```suggestion
Supports: !(uwp|linux|osx)

```

**Comment by PhoebeHui** [comment]
Let's avoid the huge changes in one PR, I think we can update that when we maintain the port, and if we still want to merge this change, we can reopen it again.

**Pull request closed without merging**